src/storage-mover/azext_storage_mover/aaz/latest/storage_mover/endpoint/_create.py

```diff
@@ -98,6 +118,32 @@ def _build_arguments_schema(cls, *args, **kwargs):
             help="The NFS protocol version.",
             enum={"NFSauto": "NFSauto", "NFSv3": "NFSv3", "NFSv4": "NFSv4"},
         )
+
+        smb_mount = cls._args_schema.smb_mount
+        smb_mount.credentials = AAZObjectArg(
+            options=["credentials"],
+            help="The Azure Key Vault secret URIs which store the required credentials to access the SMB share.",
+        )
+        smb_mount.host = AAZStrArg(
+            options=["host"],
+            help="The host name or IP address of the server exporting the file system.",
+            required=True,
+        )
+        smb_mount.share_name = AAZStrArg(
+            options=["share-name"],
+            help="The name of the SMB share being exported from the server.",
+            required=True,
+        )
+
+        credentials = cls._args_schema.smb_mount.credentials
+        credentials.password_uri = AAZStrArg(
+            options=["password-uri"],
+            help="The Azure Key Vault secret URI which stores the password. Use empty string to clean-up existing value.",
```

Comment:
Currently it seems that using an empty string is not supported by the extension:
**E.g.:**
`az storage-mover endpoint update-for-smb --resource-group $ResourceGroupName --storage-mover-name $StorageMoverName --name $SmbEndpointName --host "0.0.0.0" --share-name "dummy-share" --username-uri "" --password-uri "" --description "Cleaned-up creds"`
**Error:**
`argument --username-uri: expected one argument`

Can we support an empty string? If not, then need to make sure that we can support `null` for this, and update the documentation here accordingly.

Reply:
Still using empty string to clear the credential info. Please give it a try.

Reply:
Wondering if you are using cmd, maybe try single quotes, in Powershell both "" and '' works  
![image](https://github.com/Azure/azure-cli-extensions/assets/17182306/64107431-a530-4f6f-a5fc-193680e97e62)

Reply:
Just tried it using CMD rather than a PS session, and it worked using empty string! :)
Thank you!